arduino-ide-extension/src/browser/dialogs/ide-updater/ide-updater-dialog.tsx

```diff
@@ -130,10 +85,39 @@ export class IDEUpdaterDialog extends AbstractDialog<UpdateInfo> {
         'Software Update'
       ),
     });
+    this.node.id = 'ide-updater-dialog-container';
     this.contentNode.classList.add('ide-updater-dialog');
     this.acceptButton = undefined;
   }
 
+  private init(): void {
+    this.widget.setUpdateProgress({
+      progressInfo: undefined,
+      downloadStarted: false,
+      downloadFinished: false,
+      error: undefined,
+    });
+    if (!this.onError) {
+      this.onError = this.updaterClient.onError((error) => {
+        this.appendErrorButtons();
+        this.widget.setUpdateProgress({ error });
+      });
+    }
+    if (!this.onDownloadProgressChanged) {
+      this.onDownloadProgressChanged =
+        this.updaterClient.onDownloadProgressChanged((progressInfo) => {
+          this.widget.setUpdateProgress({ progressInfo });
+        });
+    }
+    if (!this.onDownloadFinished) {
+      // eslint-disable-next-line @typescript-eslint/no-unused-vars
+      this.onDownloadFinished = this.updaterClient.onDownloadFinished((_) => {
```

Comment:
You can use `this.onDownloadFinished = this.updaterClient.onDownloadFinished(() => {` and not `eslint-disable-*` is required.